lib/modules/manager/npm/extract/common/dependency.ts

```diff
@@ -207,16 +207,21 @@ export function getExtractedConstraints(
   deps: PackageDependency[],
 ): Record<string, string> {
   const extractedConstraints: Record<string, string> = {};
-  const constraints = ['node', 'yarn', 'npm', 'pnpm', 'vscode'];
-  for (const dep of deps) {
-    if (
-      !dep.skipReason &&
-      (dep.depType === 'engines' || dep.depType === 'packageManager') &&
-      dep.depName &&
-      constraints.includes(dep.depName) &&
-      is.string(dep.currentValue)
-    ) {
-      extractedConstraints[dep.depName] = dep.currentValue;
+  const constraints = new Set(['node', 'yarn', 'npm', 'pnpm', 'vscode']);
```

Comment:
extracting here makes no sense, they need to be extracted in post update

Reply:
Yeah, we do not these anyways, reverting.